plugins/ata/fu-ata-device.c

```diff
@@ -170,6 +218,8 @@ fu_ata_device_parse_id (FuAtaDevice *self, const guint8 *buf, gsize sz, GError *
 		return FALSE;
 	}
 
+	dell = fu_ata_device_parse_id_maybe_dell (self, id);
```

Comment:
Could we just test for `fu_device_get_guids()->len > 0`?

Reply:
sure